challenges/objects-arrays.js

```diff
@@ -49,21 +73,56 @@ const graduates = [{"id":1,"first_name":"Cynde","university":"Missouri Southern
 /* Request 1: Create a new array called universities that contains all the univeristies in the graduates array.  
 
 Once you have the new array created, sort the universities alphabetically and log the result. */
-const universities = [];
-console.log(universities)
+
+const universities = ['Missouri Southern State College', 'The School of the Art Institute of Chicago', 'Marian College', 'International Medical & Technological University',
+                      'Sultan Salahuddin Abdul Aziz Shah Polytechnic', 'Fachhochschule Rosenheim, Hochschule für Technik und Wirtschaft', 'Salem University', 'Coastal Carolina University',
+                      'Universidad Católica de Ávila', 'Universitat Rovira I Virgili Tarragona']
+ // universities.sort();
+ function sortedUni(universities){
```

Comment:
Woah! This is some nice sort algoritm. Alternatively, you could have done:
```
console.log(universities.sort())
```
You're amazing for making your own here, but there is also the `sort()` method for Arrays.

More on the sort method here: 

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/sort